packages/workflows/src/handlers/inventory/create-inventory-items.ts

```diff
@@ -15,7 +11,7 @@ export async function createInventoryItems({
   context,
   data,
 }: WorkflowArguments<{
-  products: ProductTypes.ProductDTO[]
+  inventoryItems: (InventoryItemDTO & { _associationTag?: string })[]
```

Comment:
This was a trick to eliminate the variant dependency introduced by the create product workflow. Curious to hear your thoughts @carlos-r-l-rodrigues, @adrien2p?

Reply:
I like it. It sounds more generic and reusable.